src/plugins/auth.js

```diff
@@ -38,27 +38,37 @@ export const auth = {
           const user = artifacts.decoded.payload
 
           if (!user) {
+            logger.error("Couldn't validate auth. User is missing.")
```

Comment:
We say "authorisation" in full below versus "auth" here

Shall we flip the negative and keep it simple?

Do we always put full stops in errors?

```suggestion
            logger.error('Authentication error: User missing')
```